plugins/offboard/offboard_impl.cpp

```diff
@@ -164,17 +247,19 @@ void OffboardImpl::set_velocity_ned(Offboard::VelocityNEDYaw velocity_ned_yaw)
     //const static uint16_t IGNORE_YAW = (1 << 10);
     const static uint16_t IGNORE_YAW_RATE = (1 << 11);
 
-    const float yaw = to_rad_from_deg(velocity_ned_yaw.yaw_deg);
+    _mutex.lock();
```

Comment:
Does this lock makes sure `_velocity_body_ned_yaw` has updated value?

Reply:
Right, I guess the lock prevents the race from happening. Still changed it makes more sense.